presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -188,8 +190,9 @@ public Optional<ConnectorOutputMetadata> finishCreateTable(ConnectorSession sess
     {
         JdbcOutputTableHandle handle = (JdbcOutputTableHandle) tableHandle;
         jdbcClient.commitCreateTable(session, JdbcIdentity.from(session), handle);
+        String url = baseJdbcConfig.getConnectionUrl();
```

Comment:
nit: maybe we can introduce a class parameter for URL rather than baseJdbcConfig as URL is used in multiple methods.